src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,22 @@
+package frc.team2412.robot.subsystems;
+
+import static frc.team2412.robot.Hardware.*;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class IntakeSubsystem extends SubsystemBase {
+	public static final double INTAKE_IN_SPEED = 0.3;
+	public static final double INTAKE_FAST_OUT_SPEED = -0.7;
+
+	IntakeSubsystem() {
+		CANSparkMax INTAKE_MOTOR_BACK = new CANSparkMax(INTAKE_MOTOR_BACK_ID, MotorType.kBrushless);
```

Comment:
I don't think you included all of the files you've changed/modified in your pull request

Reply:
Specifically, `INTAKE_MOTOR_BACK_ID` and the other motor constants aren't defined anywhere